src/app/pages/home/home.js

```diff
@@ -52,16 +52,12 @@ export default class Home extends BaseView {
             return;
         }
 
-        let secondaryNavHeight = appView.header.secondaryNavHeight;
-
-        if (window.pageYOffset > secondaryNavHeight && !appView.header.isPinned()) {
-            let height = appView.header.height;
-
-            appView.header.reset().collapse().pin();
-            this.el.style.paddingTop = `${height / 10}rem`;
-        } else if (window.pageYOffset <= secondaryNavHeight && !appView.header.isTransparent()) {
-            appView.header.reset().transparent();
-            this.el.style.paddingTop = '0';
+        if (window.pageYOffset >= 250) {
+            appView.header.pin().visible();
+        } else if (window.pageYOffset <= 249 && window.pageYOffset >= 150) {
```

Comment:
Why make these y offers a fixed value that is dependent on the design (and the users own settings, which can be different! — especially users with poorer vision, who can set a global CSS style to make elements and text larger)?